03_ocp_repo_sync.sh

```diff
@@ -5,6 +5,7 @@ source common.sh
 
 eval "$(go env)"
 echo "$GOPATH" | lolcat # should print $HOME/go or something like that
+export PATH="$PATH:$GOPATH/bin"
```

Comment:
Wouldn't it be better to just check that "${GOPATH}/bin" is in PATH and abort with a message if it is not, so users add it to the PATH properly in ~/.bashrc

Reply:
Given that we install golang in step 1, I assumed this was a clean machine with nothing set up. Why break the flow of running one step command after another just add something to your PATH?

Reply:
I agree, we shouldn't require anything in the user .bashrc, this looks OK to me.

In the 05 script however we reference openshift-install via an explicit path though, which is quite nice as it's very clear where the command is running from (e.g the GOPATH vs some RPM or whatever), so we could do the same here.

Reply:
This isn't working for me:

```
+ ./build.sh
I require the statik program but it's not installed.  Aborting.
You can install it with 'go get github.com/rakyll/statik'
```

However:

```
+ export PATH=/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/home/shardy/.local/bin:/home/shardy/bin:/home/shardy/go/bin
+ PATH=/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/home/shardy/.local/bin:/home/shardy/bin:/home/shardy/go/bin
+ which statik
/home/shardy/go/bin/statik
```

Reply:
The issue is GOPATH isn't exported, so it's unset in the facet build.sh, so we need to export that, not the PATH

Reply:
https://github.com/metalkube/facet/blob/master/build.sh#L3

Reply:
So, something like this?

```diff
-export PATH="$PATH:$GOPATH/bin"
+export GOPATH
```

Reply:
Or are you saying we should use `$GOPATH` in `build.sh`?

Reply:
Yeah I'm saying we already do use GOPATH in build.sh, so we need to export it or, source the go env in build.sh - maybe we should have `eval "$(go env)"` in build.sh as it's needed when any other scripts call it?